_pages/user-experience/getting-started.md

```diff
@@ -0,0 +1,80 @@
+---
+title: User Experience
+lead: >
+  Create a simple and consistent experience for your Login.gov users
+sidenav:
+  - text: Getting started
+    href: "user-experience/getting-started/"
+  - text: Sign-in and sign-out buttons
+    href: "user-experience/sign-in-sign-out/"
+  - text: Your agency logo
+    href: "user-experience/agency-logo/"
+  - text: Cancel URL
+    href: "user-experience/cancel-url/"
+  - text: Failure to proof URL
+    href: "user-experience/failure-proof/"
+  - text: Knowledge articles
+    href: "user-experience/knowledge-articles/"
+  - text: FAQ content 
+    href: "user-experience/faq-content/"
+
+---
+
+##  Getting started
+There are several decisions during the integration process that affect how your users experience Login.gov. In this User Experience guide, we will outline what options you have available to your application, and the configurations in Login.gov [dashboard](https://idp.int.identitysandbox.gov/) that impact users.
+
+{% capture item_1 %}
+[Design your application's sign-in and sign-out buttons]({{site.baseurl}}/user-experience/sign-in-sign-out/). This will include a [global sign-in button]({{site.baseurl}}/user-experience/sign-in-sign-out/), and optionally, [a sign-in page]({{site.baseurl}}/user-experience/sign-in-sign-out/) before the user is directed to Login.gov.
```

Comment:
is having the same url linked three times in one sentence ok? it feels like it's bad for accessibility.

Reply:
These will be more specific once I build the sign-in-sign-out page. It will go to specific sections in the page but I will get validation errors if I add the sections before they exist.

Reply:
got it, thanks